security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/TapPositionAction.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.action;
+
+import java.util.Objects;
+
+/**
+ * An action changing the tap position of a phase-shifting transformer.
+ *
+ * @author Hadrien Godard <hadrien.godard@artelys.com>
+ */
+public class TapPositionAction extends AbstractAction {
```

Comment:
`TapPositionAction` is not accurate enough: you have to precise that it is linked to a phase tap changer because you can have for a two windings transformer a PTC and a RTC in the same place. Maybe something like `PhaseTapChangerTapPositonAction` ?

Reply:
Yes I agree